### ISSUE TITLE
Combined pre- and post-capillary pulmonary hypertension

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6495]
**Preferred term label:** 
Combined pre- and post-capillary pulmonary hypertension
**Synonyms**
**Definition (free text, please give PubMed ID)**
Combined pre- and post-capillary pulmonary hypertension is a haemodynamic condition characterised by elevated mean pulmonary artery pressure (mPAP > 20 mmHg)  and pulmonary artery wedge pressure (PAWP >15 mmHg) and pulmonary vascular resistance (PVR ≥ 3 WU) PMID: 30545968
**Parent term (use hpo.jax.org/app)**
Elevated pulmonary artery pressure HP:0004890
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pulmonary hypertension with unclear multifactorial mechanisms ORPHA:275844
Pulmonary hypertension associated with chronic hemolytic anaemia ORPHA:275828
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik could you check the definition it seems to be the one for precapillary

**Comment by eswietlik** [comment]
@pnrobinson thank you, as before the definition ok, but the name wrong. corrected now

**Comment by pnrobinson** [comment]
added
